qemu/tests/balloon_check.py

```diff
@@ -163,10 +176,10 @@ def get_memory_boundary(self, balloon_type=''):
         if self.params.get('os_type') == 'windows':
             logging.info("Get windows miminum balloon value:")
             self.vm.balloon(1)
```

Comment:
It is used to get the min memory that windows guest can reach.  It is OK to use 1M.